docker-run

```diff
@@ -122,6 +122,7 @@ def main():
     docker_cmd.append("--env=ADDRESSES={}".format(' '.join(adrs)))
     docker_cmd.append("--env=EXTERNAL_PYTHON_VERSION_MAJOR={}".format(sys.version_info.major))
     docker_cmd.append("--env=EXTERNAL_PYTHON_VERSION_MINOR={}".format(sys.version_info.minor))
+    docker_cmd.append("--env=GEOMAPFISH_VERSION={}".format(options.version))
```

Comment:
Maybe you also want to use something like that:
https://github.com/camptocamp/c2cwsgiutils#version-information